spec/features/availability_spec.rb

```diff
@@ -27,6 +27,21 @@
     end
   end
 
+  describe 'Holdings for items with temp location', js: true do
```

Comment:
@rladdusaw there is a test in jest that needs to be updated. I would update the jest one and remove the availability_spec.rb.

in https://github.com/pulibrary/orangelight/blob/main/spec/javascript/orangelight/availability.spec.js#L548-L579
update the inner_html with:
```
<td class="holding-status" data-availability-record="true" data-record-id="99125535710106421" data-holding-id="22936525030006421" data-aeon="false" data-temp-location-code="RES_SHARE$IN_RS_REQ">
<span class="availability-icon badge badge-danger" title="Unavailable">Unavailable</span>
</td>
```
and the rest of the values accordingly.